test/system/030-run.bats

```diff
@@ -649,22 +670,41 @@ json-file | f
     run_podman tag $imageID $IMAGE
 }
 
+# bats test_tags=ci:parallel
 @test "podman inspect includes image data" {
     randomname=c_$(safename)
 
     run_podman inspect $IMAGE --format "{{.ID}}"
     expected="$IMAGE $output"
-    run_podman inspect $IMAGE --format "{{.RepoDigests}}"
-    expectedDigests="$output"
+
+    # .RepoDigests gives us an array of [quay.io/libpod/testimage@sha256:xxxx ...]
+    # .ImageDigest (below) only gives us the sha256:xxxx part.
+    run_podman inspect $IMAGE --format "{{json .RepoDigests}}"
+    # ...so, strip off the prefix and get an array of sha strings
+    declare -a expectedDigests=($(jq -r '.[]' <<<"$output" |\
+                                      sed -e "s;^${PODMAN_TEST_IMAGE_REGISTRY}/${PODMAN_TEST_IMAGE_USER}/${PODMAN_TEST_IMAGE_NAME}@;;"))
 
     run_podman run --name $randomname $IMAGE true
     run_podman container inspect $randomname --format "{{.ImageName}} {{.Image}}"
     is "$output" "$expected"
     run_podman container inspect $randomname --format "{{.ImageDigest}}"
-    assert "$output" =~ "$expectedDigests"
```

Comment:
This is the key of the problem. This test was a complete NOP.

`expectedDigests`, as explained above, is a string like `[quay.io/etc.... ]`. Note the brackets. The brackets make `=~` test for "any character in this list". Hilarity ensues.